.jshintignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
Sorry, I meant newline. But that has nothing to do with JSLint, it is so that git can properly display new changes in the future.